<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TissueTypeEditorDialog</class>
 <widget class="QDialog" name="TissueTypeEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMain">
      <attribute name="title">
       <string>&amp;Основная информация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="4" column="1">
        <widget class="QCheckBox" name="chkCounterManualInput">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblResetCounterType">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>П&amp;ериод уникальности идентификатора</string>
         </property>
         <property name="buddy">
          <cstring>cmbCounterResetType</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblCounterManualInput">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Ручной ввод идентификатора</string>
         </property>
         <property name="buddy">
          <cstring>chkCounterManualInput</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblIsRealTimeProcessing">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Обрабатывать в режиме реального времени</string>
         </property>
         <property name="buddy">
          <cstring>chkIsRealTimeProcessing</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblSex">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Пол</string>
         </property>
         <property name="buddy">
          <cstring>cmbSex</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="chkIsRealTimeProcessing">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblCounter">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>&amp;Счетчик</string>
         </property>
         <property name="buddy">
          <cstring>cmbCounter</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="cmbSex">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>М</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ж</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="CRBComboBox" name="cmbCounter">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QComboBox" name="cmbCounterResetType">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>День</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Неделя</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Месяц</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Полгода</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Год</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Постоянно</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Наименование</string>
         </property>
         <property name="buddy">
          <cstring>edtName</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edtCode"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblCode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Код</string>
         </property>
         <property name="buddy">
          <cstring>edtCode</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabIdentification">
      <attribute name="title">
       <string>&amp;Идентификация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="CInDocTableView" name="tblIdentification"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>cmbSex</tabstop>
  <tabstop>cmbCounterResetType</tabstop>
  <tabstop>chkCounterManualInput</tabstop>
  <tabstop>cmbCounter</tabstop>
  <tabstop>chkIsRealTimeProcessing</tabstop>
  <tabstop>tblIdentification</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TissueTypeEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TissueTypeEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkCounterManualInput</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbCounter</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkCounterManualInput</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblCounter</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
